packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -146,20 +146,23 @@ export const TooltipPlugin2 = ({
 
     let _plot = plot;
     let _isHovering = isHovering;
+    let _someSeriesIdx = false;
     let _isPinned = isPinned;
     let _style = style;
 
+    const updHovering = () => {
```

Comment:
nit here but similar to Kyle's comment below haha (what is wrong with just saying `updateHovering`? 😬) IMO prioritizing clarity / readability make future maintainability of this code better (i.e. not needing to question what `upd` might mean in this case vs not even needing to question this

Reply:
ok, i'll change this one :)

Reply:
![image](https://github.com/grafana/grafana/assets/22381771/17d1075b-b1db-4e56-86ba-78ad11ecd928)